ccm_web/client/src/components/ValidationErrorTable.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react'
+import { createStyles, Paper, Table, TableBody, TableCell, TableContainer, TableFooter, TableHead, TablePagination, TableRow, Theme, withStyles } from '@material-ui/core'
+import { grey } from '@material-ui/core/colors'
+import { TablePaginationActions } from './TablePagination'
+
+const StyledTableCell = withStyles((theme: Theme) =>
+  createStyles({
+    head: {
+      backgroundColor: grey[100],
+      color: theme.palette.common.black,
+      fontWeight: 'bold'
+    },
+    body: {
+      fontSize: 24
+    }
+  })
+)(TableCell)
+
+interface ValidationError {
```

Comment:
I'd consider renaming this to reference the invalid record or row; `ValidationError` sounds super generic.